documentation/markup.rst

```diff
@@ -42,7 +67,9 @@ The standard reST inline markup is quite simple: use
 
 * one asterisk: ``*text*`` for emphasis (italics),
 * two asterisks: ``**text**`` for strong emphasis (boldface), and
-* backquotes: ````text```` for code samples.
+* backquotes: ````text```` for code samples, variables, and literals.
+
+Italic is also used for function and method arguments (``*arg*``).
```

Comment:
```suggestion
```

I suggest re-eliding this from here, to avoid duplication and conflating general reST/Sphinx markdown with CPython-specific conventions